### DISCUSSION TITLE
Error messaging with details for errors in contract (eg. Error: java.lang.IllegalArgumentException: unexpected end-of-string) Error not pinpointed

### DISCUSSION THREAD

**Discussion by prateekkarmakar** [specmatic/specmatic#1499] [Ask For Help]
Following is the contract
[contract1.json](https://github.com/user-attachments/files/18250170/contract1.json)

Getting the following error while load

API Specification Summary: .\.\schemas\contract1.json
  OpenAPI Version: 3.0.1
  API Paths: 1, API Operations: 2
  Schema components: 2, Security Schemes: none
Load Error has FAILED
Reason: >> available
	      In POST /items request
	      Error: java.lang.IllegalArgumentException: unexpected end-of-string

The reason is pointing to the field but its not giving the exact description on where is the error like error is in pattern or in the description. It will be easier if the error line is available or the logs could be bit explanatory on the actual error

Using specmatic version 2.0.44

**Comment by harikrishnan83** [comment]
Thanks @prateekkarmakar for reporting this. We will try to reproduce the problem with the API spec you have shared and get back to you. cc @StarKhan6368

**Comment by StarKhan6368** [comment]
Hello @prateekkarmakar, we were able to successfully replicated the problem using the contract file you provided, We have implement a fix to improve the error descriptions, thanks!

**Comment by harikrishnan83** [comment]
Hello @prateekkarmakar , this has been resolved in [release 2.1.0](https://github.com/znsio/specmatic/releases/tag/2.1.0). Kindly have a look. I am closing this discussion for now. However please please free to reopen if you have any questions on this topic. Thanks.

cc @StarKhan6368
